src/Symfony/Component/DependencyInjection/Attribute/Target.php

```diff
@@ -18,7 +18,7 @@
  *
  * @author Nicolas Grekas <p@tchwork.com>
  */
-#[\Attribute(\Attribute::TARGET_PARAMETER)]
+#[\Attribute(\Attribute::TARGET_PARAMETER | \Attribute::TARGET_PROPERTY)]
```

Comment:
I'm not sure this PR is needed, this works already to me, see eg https://3v4l.org/W0kFq

Reply:
Seems like a PHPStorm issue then :( 
<img width="928" alt="Screenshot 2021-07-10 at 19 21 27@2x" src="https://user-images.githubusercontent.com/104180/125171343-0d64ba00-e1b4-11eb-80a5-8e971c188a36.png">